alerts/state.libsonnet

```diff
@@ -27,7 +29,9 @@
               severity: 'warning',
             },
             annotations: {
-              message: 'Ceph cluster is in warning state',
+              message: 'Storage cluster is in warning state',
+              detailed_message: 'Storage cluster is in warning state',
+              storage_type: '{{$._config.storageType}}',
```

Comment:
Should be `storage_type: $._config.storageType,`

Reply:
Ack